api/pom.xml

```diff
@@ -63,14 +63,14 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
-        <api_package>javax.enterprise.concurrent</api_package>
+        <api_package>jakarta.enterprise.concurrent</api_package>
         <non.final>false</non.final>
-        <last.spec_version>0.0</last.spec_version>
-        <next.spec_version>1.1</next.spec_version>
+        <last.spec_version>0.1</last.spec_version>
```

Comment:
Should this have been 1.1 rather than 0.1 ?

Reply:
Yes just corrected that in the latest commit